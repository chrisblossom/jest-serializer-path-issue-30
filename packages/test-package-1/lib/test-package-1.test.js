'use strict';

const testPackage1 = require('./test-package-1');
const testPackage2 = require('test-package-2');

describe('testPackage1', () => {
    test('works', () => {
        const result = {
            testPackage1: testPackage1(),
            testPackage2: testPackage2(),
        }

        expect(result).toMatchSnapshot();
    });
});
