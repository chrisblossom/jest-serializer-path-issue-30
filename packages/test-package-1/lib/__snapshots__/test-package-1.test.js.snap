// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testPackage1 works 1`] = `
Object {
  "testPackage1": "<PROJECT_ROOT>/packages/test-package-1/lib",
  "testPackage2": "<PROJECT_ROOT>/packages/test-package-2/lib",
}
`;
